popup.js

```diff
@@ -36,6 +36,17 @@ $("#ERPLoginID").blur(function() {
   }
 });
 
+
+function show_open_if_already_saved() {
+  chrome.storage.sync.get(results => {
+    chrome.extension.getBackgroundPage().console.log("results are  ", results);
```

Comment:
Do you think we can remove/comment this? I believe this is just for logging and checking in personal debugging right?

Reply:
Yes, it's okay to comment it out. Didn't notice that, thanks.